src/EmployerAccounts/SFA.DAS.EmployerAccounts.Api.UnitTests/Controllers/Reservations/WhenGettingAllReservations.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+using AutoFixture.NUnit3;
+using FluentAssertions;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.EmployerAccounts.Api.Controllers;
+using SFA.DAS.EmployerAccounts.Api.Models.Reservations;
+using SFA.DAS.EmployerAccounts.Application.Queries.GetReservations;
+using SFA.DAS.Testing.AutoFixture;
+
+namespace SFA.DAS.EmployerAccounts.Api.UnitTests.Controllers.Reservations
+{
+    public class WhenGettingAllReservations
+    {
+        [Test]
+        [MoqAutoData]
+        public async Task Then_Gets_Reservations_From_Mediator(
+            string accountId,
+            GetReservationsQueryResult mediatorResult,
+            [Frozen] Mock<IMediator> mockMediator,
+            [Greedy] ReservationController controller)
+        {
+            mockMediator
+                .Setup(mediator => mediator.Send(
+                    It.IsAny<GetReservationsQuery>(),
```

Comment:
can we check that accountId is set correctly on the mediator request here?

Reply:
done

Reply:
done